build/helper/codegen_helper.py

```diff
@@ -201,6 +202,9 @@ def _get_ctype_variable_definition_snippet_for_scalar(parameter, parameters, ivi
             definition = '{0}.{1}({2})  # case 0.0'.format(module_name, parameter['ctypes_type'], parameter['size']['value'])
         elif parameter['enum'] is not None:
             definition = '{0}.{1}({2}.value)  # case 10'.format(module_name, parameter['ctypes_type'], parameter['python_name'])
+        elif 'python_api_converter_name' in parameter:
+            converter_params = [parameter['python_name'], module_name + '.' + parameter['ctypes_type']] + parameter['python_api_converter_additional_params']
```

Comment:
I think it's simpler / clearer to get rid of additional_params if possible.
In this case, I think you could have subclasses for seconds, milliseconds, etc.

Reply:
N/A now that they are in a separate file as functions. Functions are based on seconds, etc.